ui/components/app/import-token/token-list/token-list.component.js

```diff
@@ -14,60 +30,104 @@ export default class TokenList extends Component {
     results: PropTypes.array,
     selectedTokens: PropTypes.object,
     onToggleToken: PropTypes.func,
+    currentNetwork: PropTypes.object,
+    testNetworkBackgroundColor: PropTypes.object,
   };
 
   render() {
     const {
       results = [],
       selectedTokens = {},
+
       onToggleToken,
       tokens = [],
+      currentNetwork,
+      testNetworkBackgroundColor,
     } = this.props;
 
     return results.length === 0 ? (
-      <TokenListPlaceholder />
+      <Box paddingLeft={4} paddinfRight={4} className="emptyListTokens">
+        <TokenListPlaceholder />
+      </Box>
     ) : (
-      <div className="token-list">
-        <div className="token-list__title">
-          {this.context.t('searchResults')}
-        </div>
-        <div className="token-list__tokens-container">
-          {Array(6)
+      <Box className="token-list">
+        <Box className="token-list__tokens-container">
+          {Array(12)
             .fill(undefined)
             .map((_, i) => {
               const { symbol, name, address } = results[i] || {};
               const tokenAlreadyAdded = checkExistingAddresses(address, tokens);
+              let tokenAlreadySelected;
+              if (tokenAlreadyAdded) {
+                tokenAlreadySelected = results[i];
+              }
```

Comment:
I think `tokenAlreadySelected` can be a `const` based on this snippet.  Also, can `tokenAlreadySelected` just go away?  It's used in the `isChecked` below, but can we use `tokenAlreadyAdded` there instead?